tests/Process/ProcessTest.php

```diff
@@ -426,7 +426,7 @@ public function testFakeProcessesDontThrowIfFalse()
 
     public function testRealProcessesCanHaveErrorOutput()
     {
-        if (windows_os()) {
+        if (! linux_os()) {
```

Comment:
Why have you changed this? Is this test not working on Mac?

Reply:
The core OS of Mac and Linux are not exactly the same, but they are closely related. When it comes to check os it is better to have separated test scenarios for each os to have meaningful tests.
As you can see, the message for skipping test on none windows OS is: "Requires Linux."
It's also possible to have a test scenario to check for all the Unix baced OSs and then it must have related skipping message.

Reply:
I believe this check should just use PHPUnit attribute.

Reply:
> As you can see, the message for skipping test on none windows OS is: "Requires Linux."

Yes, but the message is probably just inaccurate.

Reply:
See #50864